File: testdata/changeSignature/fromScala/AddWithDefault.java
public class SimpleMethodJava extends SimpleMethodScala {
  @Override
  public void bar(int ii, boolean b) {
    super.bar(ii, true);
  }

  public void test() {
    bar(1, true);
    SimpleMethodJava smj = new SimpleMethodJava();
    smj.bar(1, true);
  }
}

File: scala/scala-impl/testdata/refactoring/changeSignature/fromScala/AddWithDefault.scala
class SimpleMethodScala {
  def <caret>bar(ii: Int, b: Boolean): Unit = {
    ii
  }
}

class SimpleMethodChild extends SimpleMethodScala {
  override def bar(ii: Int, b: Boolean): Unit = {
    ii
    super.bar(ii, true)
  }
}